ThunderCloud/ListItem.swift

```diff
@@ -87,6 +87,10 @@ open class ListItem: StormObject, Row {
 		
 		
 		if let tableCell = cell as? TableViewCell {
+            
+            tableCell.cellTextLabel?.isHidden = title == nil || title!.isEmpty
```

Comment:
Aware the indentation is off here... I think leave it for now and at some point we should go through the whole project and re-indent each file using Xcode so we don't see any more issues when moving from tabs -> Spaces

Reply:
Title seems to be force-unwrapped here?